### ISSUE TITLE
Replacement transaction error

### ISSUE THREAD

**Issue by ceseale** [polyswarm/polyswarmd#97]
**Describe the bug**

> .. I was getting nonce errors before staking, staked (which failed with 'replacement transaction underpriced'), but then it started working again

**Expected behavior**
Needs investigation, might be a nonce issue.

What does the error mean?

> "Returned error: replacement transaction underpriced"

The error means that:
1. You have a pending transaction from your account in your Ethereum client
1. The new transaction you are sending has the same nonce as that pending transaction
1. The new transaction you sent has a gas price that is too small to replace the pending transaction

**Additional Information**
https://stackoverflow.com/questions/46349202/rinkeby-replacement-transaction-underpriced

**Comment by mjkoo** [comment]
Yes that is a nonce issue, after that error it should have resynced and recovered?
